### ISSUE TITLE
Feature request: Requester Pays

### ISSUE THREAD

**Issue by benyanke** [awslabs/aws-js-s3-explorer#41]
It would be great to allow the requests to be sent with requester pays.

**Comment by john-aws** [comment]
Yes,this sounds like a useful feature. Thanks for your work here. Will review once complete.

**Comment by benyanke** [comment]
Excellent. I have a proof of concept mostly working in #43  without the config option (aka - hardcoded to use requester pays).

I'll be testing that, and then integrating with config.
